.env

```diff
@@ -0,0 +1,2 @@
+PORT=8080
+JWT_SECRET=c6dfe9243c6a50994a7b252c9901cf91233371103a3c319e6e8c3116aceca54cdf9122ac1a40a44a697cb7e8432019c34f2e6a317d6f3350a352aff0a3b5fe69
```

Comment:
.env file should not be saved on GitHub because you share the sensitive values will everybody who has an access to this repo